lib/commands/console.cmd.js

```diff
@@ -35,6 +35,11 @@ module.exports = () => {
       alias: 'fdt',
       describe: 'Will list all production licenses for all modules.',
       default: false
+    },
+    'production-only': {
```

Comment:
If you think this should be the default behaviour, but still configurable by an option, we could change it to `--exclude-devdependencies` or something?

Reply:
Also using the alias 'edd' 
`--exclude-dev-dependencies` or `--edd` just to follow the pattern of the others

Reply:
Looking great, and yes, if the default behavior is to use production only it will save the users to add 1 more option , making the command call more clean. +1